resources/views/menu/admin_menu.blade.php

```diff
@@ -0,0 +1,72 @@
+<ul class="nav navbar-nav">
+
+@foreach ($items->sortBy('order') as $item)
+    
+    @php
+        // TODO - still a bit ugly - can move some of this stuff off to a helper in the future.
+        $listItemClass = [];
+        $styles = null;
+        $linkAttributes = null;
+
+        if(url($item->url) == url()->current()){
```

Comment:
Please use `$item->link()` instead of `$item->url`.
Sometimes `url` is empty since they might use `route` instead.
But `$item->link()` will check for that and return the correct link url.